### PR TITLE
filter-grid

### PR DESCRIPTION
**Pull request by Teruah2540**
Initial filter action grid documentation

### CODE REVIEW

File: docs/actions/filter-grid.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## `Input Parameter Reference`
+
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| Select Module | Select the Grid Module you wish to filter| Yes | None Selected| No |
+| Page Size | Select the number of elements(rows) that will be shown in grid| No | None | No |
+| Sort By | Select the field to sort by. Ascending is the default but you can select Descending using the checkbox| Yes | Ascending | No |
+| Search Terms| Grid will automatically search for the characters passed by this field| Yes | Unchecked | No |
+
```
Comment (pande3) [inline docs/actions/filter-grid.md:25]: Even though they aren't included on the current DNN Sharp documentation, I think we should go ahead and add the **Current Page**, **Filters**, **Replace Filters**, and **Additional Data** parameters here.

**Review by pande3 — COMMENTED** [review]
Hi Linda, 

I added some comments below! Cool that you were able to open the PR!

-Patrick

File: docs/actions/filter-grid.md (hunk 1 above)
Comment (pande3) [reply docs/actions/filter-grid.md:27]: Hi Linda, 

This review process is frustrating, I know! I can't tell whether you added these additional params or not. If not, I can add them, but I don't want to redo something you already did or may be in process. 

Let me know, 
-Patrick

File: docs/actions/filter-grid.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| Select Module | Select the Grid Module you wish to filter| No | None Selected| No |
+
+| Page Size | Select the number of elements(rows) that will be shown in grid| Yes | None | No |
+
+| Sort By | Select the field to sort by. Ascending is the default but you can select Descending using the checkbox| Yes | Ascending | No |
+| Search Terms| Grid will automatically search for the characters passed by this field| Yes | Unchecked | No |
+
```
Comment (Teruah2540) [inline docs/actions/filter-grid.md:27]:
```suggestion

| Current Page | Leave empty to keep current page | Yes | Empty String | No |
| Filters | Allows the grid to be filtered using multiple criteria | Yes | None | No |
| Replace Filters | By default, filters are merged with existing filtering, if any. Use this option to replace all filters | No | Unchecked| No |
| Additional Data | Provide additional values that will be passed the grid via query string. You can reference them using [$] syntax.  | Yes | None | No |

```

**Review by Teruah2540 — COMMENTED** [review]
Added Current page, filters, replace filters and additional data

File: docs/actions/filter-grid.md (hunk 1 above)
Comment (Teruah2540) [reply docs/actions/filter-grid.md:29]: @pande3 I believe I have added the additional parameters so you can see them now.  Let me know if it isn't right.

File: docs/actions/filter-grid.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| Select Module | Select the Grid Module you wish to filter| No | None Selected| No |
+| Page Size | Select the number of elements(rows) that will be shown in grid| Yes | None | No |
+| Sort By | Select the field to sort by. Ascending is the default but you can select Descending using the checkbox| Yes | Ascending | No |
+| Search Terms| Grid will automatically search for the characters passed by this field| Yes | Unchecked | No |
+| Current Page | Leave empty to keep current page | Yes | Empty String | No |
+| Filters | Allows the grid to be filtered using multiple criteria | Yes | None | No |
+| Replace Filters | By default, filters are merged with existing filtering, if any. Use this option to replace all filters | No | Unchecked| No |
+| Additional Data | Provide additional values that will be passed the grid via query string. You can reference them using [$] syntax.  | Yes | None | No |
```
Comment (pande3) [inline docs/actions/filter-grid.md:28]:
```suggestion
| Select Module | Select the Grid Module you wish to filter| No | None Selected| No |
| Current Page | Leave empty to keep current page | Yes | Empty String | No |
| Page Size | Select the number of elements(rows) that will be shown in grid| Yes | None | No |
| Sort By | Select the field to sort by. Ascending is the default but you can select Descending using the checkbox| Yes | Ascending | No |
| Search Terms| Grid will automatically search for the characters passed by this field| Yes | Unchecked | No |
| Filters | Allows the grid to be filtered using multiple criteria | Yes | None | No |
| Replace Filters | By default, filters are merged with existing filtering, if any. Use this option to replace all filters | No | Unchecked| No |
| Additional Data | Provide additional values that will be passed to the grid via query string. You can reference them using [$] syntax.  | Yes | None | No |

```

Comment (pande3) [reply docs/actions/filter-grid.md:28]: I changed the order of the items to match the order they occur in the action settings and added the word "to" in the last setting.

**Pull request merged**